### PR TITLE
Automated cherry pick of #44169 release 1.5

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -859,6 +859,9 @@ function start-kube-apiserver {
   params+=" --secure-port=443"
   params+=" --tls-cert-file=${APISERVER_SERVER_CERT_PATH}"
   params+=" --tls-private-key-file=${APISERVER_SERVER_KEY_PATH}"
+  if [[ -n "${SERVICEACCOUNT_CERT_PATH:-}" ]]; then
+    params+=" --service-account-key-file=${SERVICEACCOUNT_CERT_PATH}"
```
Comment (liggitt) [inline cluster/gce/gci/configure-helper.sh]: SERVICEACCOUNT_KEY_PATH?

**Comment by mikedanese** [comment]
This fixes #44285

File: cluster/gce/gci/configure-helper.sh (hunk 1 above)
Comment (mikedanese) [inline cluster/gce/gci/configure-helper.sh]: Either works.

**Pull request merged**